packages/components/src/ResourcePicker/Toolbar/NameFilter/NameFilter.component.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import DebounceInput from 'react-debounce-input';
+
+import theme from './NameFilter.scss';
+
+function NameFilter({ label, placeholder, value, debounceTimeout, onChange }) {
+	return (
+		<form className={classNames('name-filter', theme['name-filter'])}>
+			{label && (
```

Comment:
👏 Good job

Reply:
It means that if label and placeholder are not provided, it is not accessible.
The label is hidden, so perhaps only label or placeholder could be in the component api, and the value is set on both.
And you can make it required.